src/org/usfirst/frc/team2175/driverstation/DriverStation.java

```diff
@@ -114,7 +114,7 @@ public double getTurnValue() {
     }
 
     public double getClimberSpinSpeed() {
-        return gamepad.getRawAxis(1);
+        return gamepad.getRawAxis(1) / 2;
```

Comment:
Instead of a magic number here, it would be great if this was configurable from a property file.